### DISCUSSION TITLE
There is no option to cap the frame rate at display refresh rate without Vsync

### DISCUSSION THREAD

**Discussion by Duckleeng** [ppy/osu#23161] [Ideas]
There is no option to cap the frame rate at display refresh rate without Vsync, and using Vsync comes with a latency penalty.
An option to cap at 1x Refresh Rate should definitely be added.

**Comment by Eason0729** [comment]
I agree with the idea of adding ``1x refresh rate``, but not because of latency penalty. 
I installed osu! from flathub on KDE Neon recently, and I could only reach half of the frame rate under Vsync. (This is probably a bug in either my configuration or KDE Neon)
Anyway, I just set the option to ``2x refresh rate`` and ignore the bug. But it would be nice to have ``1x refresh rate`` to conserve my laptop's battery life.

**Reply by Duckleeng** [reply]
Wdym not because of latency penalty? You want more latency?

**Reply by Eason0729** [reply]
Lower latency is certainly one of the reason, but I care more about the power consumption of my laptop.

**Reply by Eason0729** [reply]
By the way, I have moved to Fedora since it's only 3 days after I installed KDE Neon.So my problem was solved(partically) on Fedora.
